src/base32.c

```diff
@@ -322,7 +330,15 @@ char* Base32_Encode_Bytes(const unsigned char* source, size_t size)
     {
         /* Codes_SRS_BASE32_07_005: [ If size is 0 Base32_Encode shall return an empty string. ] */
         result = malloc(1);
-        strcpy(result, "");
+        if (result == NULL)
+        {
+            /* Codes_SRS_BASE32_07_014: [ Upon failure Base32_Encode_Bytes shall return NULL. ] */
+            LogError("encoding for zero-size input failed.");
```

Comment:
is this not malloc error?

Reply:
yes, it's a malloc error, but i was trying to align the error-log with the other ones in the same function.